layer/CMakeLists.txt

```diff
@@ -78,32 +78,29 @@ endif()
 
 add_vk_layer(${TARGET_NAME} profiles.cpp vk_layer_table.cpp ${jsoncpp_generated_source} ${jsoncpp_generated_header})
 
-if (REGENERATE_LAYER)
-    set(LAYER_PYTHON_FILES ${CMAKE_SOURCE_DIR}/scripts/gen_profiles_layer.py)
-    source_group("Python Files" FILES ${LAYER_PYTHON_FILES})
-
-    add_custom_target(VpLayer_generate ALL
-        COMMAND ${PYTHON_EXECUTABLE} ${LAYER_PYTHON_FILES}
-            -registry ${vulkan-headers_SOURCE_DIR}/registry/vk.xml
-            -outLayer ${CMAKE_SOURCE_DIR}/layer/profiles.cpp
-        VERBATIM
-        SOURCES ${LAYER_PYTHON_FILES}
-        DEPENDS ${vulkan-headers_SOURCE_DIR}/registry/vk.xml)
-    set_target_properties(VpLayer_generate PROPERTIES FOLDER "Profiles layer")
-    add_dependencies(VkLayer_${TARGET_NAME} VpLayer_generate)
+set(LAYER_PYTHON_FILES ${CMAKE_SOURCE_DIR}/scripts/gen_profiles_layer.py)
```

Comment:
Why have we removed `REGENERATE_LAYER`?

Reply:
All the option didn't actually work and in any case we need to build the layer.